lib/src/migrators/module.dart

```diff
@@ -1228,11 +1252,16 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
   ///
   /// Otherwise, returns [name] unaltered.
   String _unprefix(String name) {
-    var isPrivate = name.startsWith('-');
-    var unprivateName = isPrivate ? name.substring(1) : name;
+    var isPrivate = _isPrivate(name);
+    var unprivateName = isPrivate ? _privateToPublic(name) : name;
     var prefix = _prefixFor(unprivateName);
     if (prefix == null) return name;
-    return (isPrivate ? '-' : '') + unprivateName.substring(prefix.length);
+
+    var withoutPrefix = unprivateName.substring(prefix.length);
+    if (_isPrivate(withoutPrefix)) {
+      withoutPrefix = _privateToPublic(withoutPrefix);
```

Comment:
In this case maybe it's better to only run substring(1), otherwise multiple dashes will get replaced to 1 dash still

Reply:
Why? `_privateToPublic()` should handle multiple dashes gracefully.